deploy/flux-installer/templates/apps/istio/istiod-repo.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This is a duplicate of the `istio-base` repo, all of istio can use one HelmRepository resource.